.github/actions/nr-verify/index.js

```diff
@@ -0,0 +1,63 @@
+import { args } from './args.js'
+import { fetchRetry } from '../shared-utils/fetch-retry.js'
+import { constructLoaderFileNames, constructFuzzyVersions } from '../shared-utils/loaders.js'
+
+const loaderFileNames = constructLoaderFileNames(args.loaderVersion)
+const fuzzyVersions = constructFuzzyVersions(args.loaderVersion)
+const envOptions = {
+  stage: {
+    url: 'https://staging-api.newrelic.com/v2/js_agent_loaders/version.json'
+  },
+  prod: {
+    url: 'https://api.newrelic.com/v2/js_agent_loaders/version.json'
+  }
+}
+
+const verifyJobs = args.environment.map(env => {
+  return loaderFileNames.map(loaderFileName => {
+    return [
+      {
+        env,
+        loaderFileName
+      },
+      {
+        env,
+        loaderFileName: loaderFileName.replace(args.loaderVersion, fuzzyVersions.PATCH)
+      },
+      {
+        env,
+        loaderFileName: loaderFileName.replace(args.loaderVersion, fuzzyVersions.MINOR)
+      }
+    ]
+  }).flat()
+}).flat()
+
+const results = await Promise.allSettled(
```

Comment:
Should check with API-E hero to see if there are any concerns with rate limiting.